packages/core/src/components/VerticalNav/VerticalNav.scss

```diff
@@ -1,5 +1,19 @@
 @import '@cmsgov/design-system-support/src/index';
 
+%nav-item-hover {
```

Comment:
Previously the hover would override the focus, but thats not consistent with how focus works on `Tab` and other components. This commit makes the focus work the same across those components